projects/core/src/cart/facade/cart-validation.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { combineLatest, Observable } from 'rxjs';
+import { CartValidationConnector } from '../connectors/validation/cart-validation.connector';
+import { CartModificationList } from '../../model/cart.model';
+import { switchMap } from 'rxjs/operators';
+import { Command, CommandService, CommandStrategy } from '../../util';
+import { UserIdService } from '../../auth';
+import { ActiveCartService } from './active-cart.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CartValidationService {
+  protected getCartModificationListCommand: Command = this.command.create(
+    () =>
+      combineLatest([
+        this.activeCartService.getActiveCartId(),
+        this.userIdService.takeUserId(),
+      ]).pipe(
```

Comment:
To be safe that we execute it only once I recommend using take(1)
```suggestion
      combineLatest([
        this.activeCartService.getActiveCartId(),
        this.userIdService.takeUserId(),
      ]).pipe(
      	take(1),
```